### PR TITLE
Wu feature email verification - feature implemented, pending testing

### PR DESCRIPTION
**Pull request by qingyuan-wu**

### CODE REVIEW

**Comment by qingyuan-wu** [comment]
Modify logins schema that contains info regarding whether the user is "verified":
```sql
DROP TABLE IF EXISTS `logins`;
CREATE TABLE `logins` (
  `id` int NOT NULL AUTO_INCREMENT,
  `first-name` text NOT NULL,
  `last-name` text NOT NULL,
  `email` text NOT NULL,
  `password` text NOT NULL,
  `verified` TINYINT(1) DEFAULT 0,
  PRIMARY KEY (`id`)
); 
```

**Comment by qingyuan-wu** [comment]
Add new table `email_verification` that stores the `uuid` required to verify new users.
```sql
DROP TABLE IF EXISTS `email_verification`;
CREATE TABLE `email_verification` (
  `email` VARCHAR(200) NOT NULL,
  `first-name` text NOT NULL,
  `last-name` text NOT NULL,
  `uuid` text NOT NULL,
  PRIMARY KEY (`email`)
); 
```

File: config/mail-transporter.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const nodemailer = require("nodemailer");
+
+let transporter = nodemailer.createTransport({
+	service:"Yandex",
+	host: "smtp.yandex.ru",
```
Comment (harsimrat99) [inline config/mail-transporter.js]: Can we move this to an env. var?

**Comment by qingyuan-wu** [comment]
I have added the middleware for email verification. This middleware is triggered whenever the user attempts to access the new post page (GET), post a new post (POST), or edit their profile (POST). The user is still able to get their profile, but they will be prompted to verify their email instead of editing it.

Qingyuan

**Review by harsimrat99 — APPROVED** [review]

**Comment by harsimrat99** [comment]
Hi Qingyuan,

Thank you for making the requested changes and for your work in driving this feature. I will merge this branch and close this PR.

Best,
Harsimrat

**Pull request merged**